nipype/algorithms/rapidart.py

```diff
@@ -665,14 +534,122 @@ def _run_interface(self, runtime):
                 nrows.append(mc_in.shape[0])
             matrix = self._get_spm_submatrix(spmmat, sessidx, rows)
             self._stimcorr_core(motparamlist[i], intensityfiles[i],
```

Comment:
Codacy only found this?? Then it's useless! :P

Seriously, this is still very preliminary. I am coding with the pylint on, so I also address these issues. I'll let you know when this is ready.

BTW, with this refactoring, we get rid of the infamous ```_list_outputs``` :)